test/support.js

```diff
@@ -29,7 +29,7 @@ Sequelize.Promise.onPossiblyUnhandledRejection(function(e, promise) {
 Sequelize.Promise.longStackTraces();
 
 // shim all Sequelize methods for testing for correct `options.logging` passing
-if (!process.env.COVERAGE) supportShim(Sequelize);
+if (!process.env.COVERAGE && false) supportShim(Sequelize);
```

Comment:
H @mickhansen. Work has quietened down now, so I've got some time to get back on coding. Was planning to finally get round to writing the tests for transaction and CLS context passing.

How did the supportShim fuck up? Happy to have a look and figure it out...